src/FormBuilder/Actions/DequeueAdminScriptsInFormBuilder.php

```diff
@@ -22,6 +22,8 @@ public function __invoke()
         if ($this->isFormBuilderRoute()) {
             $wpScripts = wp_scripts();
 
+            $wpScriptsRegistered = array_column($wpScripts->registered, 'handle');
+
```

Comment:
(from merge) This can be removed

Reply:
Resolved, see https://github.com/impress-org/givewp-next-gen/pull/181/commits/01b226ccb13828fd75e51437234609c4cf70c562.